neqo-transport/src/tparams.rs

```diff
@@ -19,6 +18,7 @@ use neqo_crypto::{
     ext::{ExtensionHandler, ExtensionHandlerResult, ExtensionWriterResult},
     random, HandshakeMessage, ZeroRttCheckResult, ZeroRttChecker,
 };
+use rustc_hash::FxHashMap as HashMap;
```

Comment:
This is used for our transport parameters, which is another place where an `EnumMap` could save us time.  We currently key on a `u64`, but we don't allow inserts unless we understand the transport parameter, so it would be possible to switch.